helpers/string.py

```diff
@@ -0,0 +1,54 @@
+from enum import Enum
+from typing import List
+
+
+class EscapeEnum(Enum):
+    APPEND = "append"
+    PREPEND = "prepend"
+    REPLACE = "replace"
+
+
+class Replacement:
```

Comment:
[nit]
I believe you can use a [@dataclass](https://docs.python.org/3/library/dataclasses.html) for this. That way you don't have to specify the `__init__` function, just the fields.

```python
@dataclass
class Replacement:
    chars: str
    output: str
    method: EscapeEnum
```